go-controller/pkg/allocator/pod/pod_annotation.go

```diff
@@ -423,6 +434,11 @@ func allocatePodAnnotationWithRollback(
 			if err != nil {
 				return
 			}
+		} else if hasIPAMClaim {
+			tentative.IPs, err = util.ParseIPNets(ipamClaim.Status.IPs)
+			if err != nil {
```

Comment:
Don't we want to at least print the error ?

Reply:
I'm just repeating the pattern around here; please check 3/4 lines above - if we fail when parsing the IP addresses from the user provided info - `util.ParseIPNets` - we also do not log anything.